### ISSUE TITLE
USTVGO is now working without VPN

### ISSUE THREAD

**Issue by JayDee696969** [Tempest0580/Temptv#67]
So I noticed Apex Sports addon has a working USTVGO inside the addon under Live TV. It seems there is no VPN blocking with this version. Maybe you can take a look and see how they did it? https://github.com/arxtic-ape/repo.arxtic-ape

**Comment by simakuutio** [comment]
I can confirm this. Right now comparing temptv code with that another to see if they have found some universal solution... :) At least that another working code is _much_ simplier than temptv... and it have different approach also. This needs to be evaluated properly. Very good catch, thank you!

**Comment by wolfhead37** [comment]
that's ustvgo,not usatvgo. I am not sure but prob two different things.

**Comment by wolfhead37** [comment]
also ustvgo works with a USA vpn.

**Comment by bjgood** [comment]
No. It's the same ustvgo.tv that's in TempTV...that's for sure. I'm looking the code over right now. I think I see why this one works and TempTV doesn't. Now to do a little experimenting.
Later...

**Comment by simakuutio** [comment]
> No. It's the same ustvgo.tv that's in TempTV...that's for sure. I'm looking the code over right now. I think I see why this one works and TempTV doesn't. Now to do a little experimenting.

Yep, big difference is that requests.post which returns stream url. TempTV approach is somewhat different. Maybe TempTV author should contact that another and and ask if that new approach could be merged into TempTV (it's always a good idea to ask permission). Naturally, similar approach can be easily done manually too.

**Comment by wolfhead37** [comment]
OK, i just saw diff name and icon,  so I assumed it was something different, but yeah the channel setup is exactly the same. seems to be just as stable, too, used if for a few hours last night with no trouble, connected to USA VPN.

**Comment by JayDee696969** [comment]
Yep, but i do prefer the channel icons that TempTV provides, so hopefully he can look it over and implement it.
